alexandra/__init__.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
This tells flake8 to ignore the whole file, because otherwise it'd complain about unused imports